MBProgressHUD.m

```diff
@@ -220,15 +220,16 @@ - (void)showUsingAnimation:(BOOL)animated {
 
 - (void)hideUsingAnimation:(BOOL)animated {
     if (animated && self.showStarted) {
+        self.showStarted = nil;
         [self animateIn:NO withType:self.animationType completion:^(BOOL finished) {
             [self done];
         }];
     } else {
+        self.showStarted = nil;
```

Comment:
Could be consolidated to just one entry above `f (animated && self.showStarted) {`.